packages/client/src/client/App/Footer/ContactUsButton/FollowUs.tsx

```diff
@@ -17,10 +16,10 @@ const Row = ({
       variant="Text sm/Regular underlined"
       color="Component colors/Utility/Blue dark/utility-blue-dark-500"
       onClick={onClick}
+      paddingBottom="md"
```

Comment:
^ ^ can we get some typing on the color, do you think?
I know it has to allow "red", "white" and "inherit" and wotnot .. so not that easy .. but perhaps "csstype"'s Color | UISK theme Color ?
.. seems the base `div` component's def of `color` takes precedence - may not be worth the effort, but it would be  nice to have completion, I would think ..